CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## Unreleased
+
+### Features
+- Send react native js engine, turbo module, fabric flags and component stack in Event contexts ([#2552](https://github.com/getsentry/sentry-react-native/pull/2552))
```

Comment:
```suggestion
### Features

- Send react native js engine, turbo module, fabric flags and component stack in Event contexts ([#2552](https://github.com/getsentry/sentry-react-native/pull/2552))
```

@krystofwoldrich always jump a line, otherwise, it gets ugly.